CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
   figure.
 * Fixed issue with Speck not rendering unless it is attached to a callback.
 * Prevent Speck from trying to calculate a system with no atom.
+* Let Clustergram render even when default row and column labels are not 
+  specified by adding default values. 
```

Comment:
```suggestion
  specified, by adding default values.
```

Reply:
I'm suggesting this additional comma because the following confusion could arise: "labels are not specified by adding default values" vs "let Clustergram render by adding default values" (and we mean the latter).